project/Dependencies.scala

```diff
@@ -69,9 +69,9 @@ object Dependencies {
     commonDeps ++ jacksonOverrides ++ akkaSerializationJacksonOverrides ++ Seq(
       evolutions,
       jdbc,
-      "com.gu.play-googleauth" %% "play-v28" % "2.2.7",
-      "com.gu.play-secret-rotation" %% "play-v28" % "0.38",
-      "com.gu.play-secret-rotation" %% "aws-parameterstore-sdk-v2" % "0.38",
+      "com.gu.play-googleauth" %% "play-v28" % "2.2.8",
+      "com.gu.play-secret-rotation" %% "play-v28" % "0.40",
```

Comment:
Elsewhere, in `project/plugins.sbt`, this project is getting upgraded to Play v2.9, so the versions of `play-googleauth` & `play-secret-rotation` need to change too:

```suggestion
      "com.gu.play-googleauth" %% "play-v29" % "2.2.8",
      "com.gu.play-secret-rotation" %% "play-v29" % "0.40",
```

...you can see that otherwise, the PR build [fails](https://github.com/guardian/riff-raff/actions/runs/6878989852/job/18709910006#step:6:268) with the exciting message:


> [error] /home/runner/work/riff-raff/riff-raff/riff-raff/app/AppComponents.scala:248:3: Symbol 'term play.api.BuiltInComponentsFromContext.configuration' is missing from the classpath.
> [error] This symbol is required by ' <none>'.
> [error] Make sure that term configuration is in your classpath and check for conflicting dependencies with `-Ylog-classpath`.
> [error] A full rebuild may help **if 'RotatingSecretComponents.class' was compiled against an incompatible version of play.api.BuiltInComponentsFromContext**.
> [error]   context.lifecycle.addStopHook(() =>
> [error]   ^